ci/gpu/build.sh

```diff
@@ -98,9 +98,13 @@ fi
 # TEST - Run GoogleTest and py.tests for libcugraph and cuGraph
 ################################################################################
 
-set +e -Eo pipefail
+# Switch to +e to allow failing commands to continue the script, which is needed
+# so all testing commands run regardless of pass/fail. This requires the desired
+# exit code to be managed manually by updating it only for commands that
+# represent test failures - other less critical commands can fail if necessary
+# (nbtestlog2junitxml, codecov, etc.) to keep CI unblocked.
```

Comment:
Hm. I know we talked about removing the `trap` here, but now I'm thinking it might be better to keep it. I think we'd be setting ourselves up for some potential issues in the future if we're not trapping all errors when `set +e` is set. Using the `trap` would make this behave more closely to `set -e`, where all erroneous exit codes are accounted for (instead of just specific ones).

Can you re-add the `trap` logic here?

Reply:
sure I'll add it back in the next commit.